### ISSUE TITLE
Chage Docker base-image to Ubuntu 22.04

### ISSUE THREAD

**Issue by hirschnase** [Maescool/docker-Cloudlog#5]
I'd like to suggest to change the Ubuntu base image to 22.04. I experienced some trouble with the reachability of some of the ubuntu Archive servers which went away after chaning the base to 22.04. Also, it's newer and longer supported.

Necessary changes to the Dockerfile are only two lines (line 1 and line 9):

```
FROM ubuntu:22.04

ENV PHPVER 7.4

# Update and install ubuntu packages
RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get -y dist-upgrade && \
    DEBIAN_FRONTEND=noninteractive apt-get -y install software-properties-common locales

RUN add-apt-repository ppa:ondrej/php --yes && apt update

RUN DEBIAN_FRONTEND=noninteractive apt-get -y install supervisor cron nginx php$PHPVER php$PHPVER-cli php$PHPVER-fpm php$PHPVER-mbstring php$PHPVER-xml php$PHPVER-curl php$PHPVER-gd php$PHPVER-mysql php-redis php$PHPVER-readline mariadb-client wget git unzip curl vim
```

**Comment by Wireheadbe** [comment]
Going to:
```
FROM ubuntu:22.04

ENV PHPVER 8.1
```
Doesn't require any out-of-tree PPA's.

**Comment by Maescool** [comment]
Fixed and merged in pull #8
